server/controllers/users.js

```diff
@@ -310,4 +310,44 @@ export default class User {
       .catch(() => res.status(500).json({ error: 'Server error' }));
     return this;
   }
+  /**
+ *
+ *
+ * @param {any} req
+ * @param {any} res
+ * @memberof User
+ */
+  changePassword(req, res) {
+    const { oldPassword } = req.body;
```

Comment:
### Nitpick

You can unpack this and the line below on just one line.
`const { oldPassword, password } = req.body;`